### ISSUE TITLE
Selected element (date) is not properly identified

### ISSUE THREAD

**Issue by Gabrielaparada** [duetds/date-picker#129]
The dates have an attribute of 'aria-pressed' which updates to true/false as they are selected/unselected. However, this is not the correct attribute to communicate the selected/unselected date, aria-pressed should only be used for toggle buttons which is not the case in this scenario. 

**Expected behavior**
aria-pressed should be replaced with 'aria-selected'

<img width="500" alt="Screenshot 2024-05-08 at 1 26 43 PM" src="https://github.com/duetds/date-picker/assets/53956875/6781eea7-374e-448f-8e13-45c99283573c">

**Comment by jonnystening** [comment]
Check out these previous changes …

- https://github.com/duetds/date-picker/pull/91
- https://github.com/duetds/date-picker/pull/51

Saw this issue come through and remembered addressing this a while back, haven’t looked into it, sharing in case of interest

**Comment by WickyNilliams** [comment]
`aria-selected` is not valid on tables. Only on `role=grid`. We actually started out with that, but grid has its own set of problems.

After discussion with Adrian Roselli, we settled on the current approach. See https://github.com/duetds/date-picker/issues/26

Do you have any further info on `aria-pressed` being incorrect?

In any case, this repo is for all intents dead. I no longer have commit access, so any changes are unlikely. I have created a spiritual successor of sorts, [Cally] (https://github.com/WickyNilliams/cally). I'd be happy to continue discussion there if you like, and see what can be done
